libs/moloch-v3-macro-ui/src/components/DaoSettings/GovernanceSettings.tsx

```diff
@@ -0,0 +1,194 @@
+      quorum: dao.quorumPercent,
+      minRetention: dao.minRetentionPercent,
+      sponsorThreshold: toWholeUnits(dao.sponsorThreshold),
+      newOffering: toWholeUnits(dao.proposalOffering),
+      vStake: dao.sharesPaused,
+      nvStake: dao.lootPaused,
+    };
+  }, [dao]);
+
+  return (
+    <GovernanceContainer>
+      <GovernanceCardHeader>
+        <H3>Governance Settings</H3>
+        {includeLinks && (
+          <ButtonRouterLink
+            color="secondary"
+            to={`/molochv3/${daoChain}/${
+              dao.id
+            }/new-proposal?formLego=UPDATE_GOV_SETTINGS&defaultValues=${JSON.stringify(
+              defaultValues
+            )}`}
+          >
+            Update DAO Settings
+          </ButtonRouterLink>
+        )}
+      </GovernanceCardHeader>
+      <div className="description">
+        <ParSm>
+          <StyledLink
+            href="https://moloch.daohaus.fun/configuration/governance-configuration"
+            target="_blank"
+            rel="noreferrer"
+          >
+            Review the documenation
+          </StyledLink>{' '}
+          for additional details on governance settings. Updates to settings
+          will go through a proposal.
+        </ParSm>
+      </div>
+      <DataGrid>
+        <DataIndicator
+          size="sm"
+          label="Voting Period"
+          data={formatPeriods(dao.votingPeriod)}
+          info={INFO_COPY.VOTING_PERIOD}
+        />
+        <DataIndicator
+          size="sm"
+          label="Grace Period"
+          data={formatPeriods(dao.gracePeriod)}
+          info={INFO_COPY.GRACE_PERIOD}
+        />
+        <DataIndicator
+          size="sm"
+          label="New Offering"
+          data={`${fromWei(dao.proposalOffering)} ${networkData?.symbol}`}
+          info={INFO_COPY.NEW_OFFERING}
+        />
+      </DataGrid>
+      <DataGrid>
+        <DataIndicator
+          size="sm"
+          label="Quorum %"
+          data={formatValueTo({ value: dao.quorumPercent, format: 'percent' })}
+          info={INFO_COPY.QUORUM}
+        />
+        <DataIndicator
+          size="sm"
+          label="Min Retention %"
+          data={formatValueTo({
+            value: dao.minRetentionPercent,
+            format: 'percent',
+          })}
+          info={INFO_COPY.MIN_RETENTION}
+        />
+        <DataIndicator
+          size="sm"
+          label="Sponsor Threshold"
+          data={`${fromWei(dao.sponsorThreshold)} Voting Tokens`}
+          info={INFO_COPY.SPONSOR_THRESHOLD}
+        />
+      </DataGrid>
+      <TokensHeader>
+        <H3 className="tokens">DAO Tokens</H3>
+        {includeLinks && (
+          <ButtonRouterLink
+            color="secondary"
+            to={`/molochv3/${daoChain}/${
+              dao.id
+            }/new-proposal?formLego=TOKEN_SETTINGS&defaultValues=${JSON.stringify(
+              defaultValues
+            )}`}
+          >
+            <Button color="secondary">Update Token Settings</Button>
```

Comment:
this is throwing the following warning:

<img width="536" alt="image" src="https://user-images.githubusercontent.com/926341/219484869-70ad00ba-b7e5-4bc9-8167-5c9156aaf8dc.png">